src/server/api/inbox.ts

```diff
@@ -134,6 +134,11 @@ export const inboxRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubS
     }
 
     await apsystem.approveActivity(actor, id)
+    const activity = await store.forActor(actor).inbox.get(id)
+    // Add to ReplyStore if the activity is a reply
+    if (activity.type === 'Reply') {
```

Comment:
I think the type should be "create" and the object is a type: Note with a property called `inReplyTo`

Check the docs here: https://blog.joinmastodon.org/2018/06/how-to-implement-a-basic-activitypub-server/ and here: https://www.w3.org/wiki/ActivityPub/Primer/Threading